README.md

```diff
@@ -1,26 +1,35 @@
-[![](https://jitpack.io/v/spotify/openfeature-kotlin-sdk.svg)](https://jitpack.io/#spotify/openfeature-kotlin-sdk)
```

Comment:
Do we want to keep this?

Reply:
Good point, I don't think it does anything. Will remove it.